anypoint-mq/4.x/modules/ROOT/pages/anypoint-mq-failover.adoc

```diff
@@ -21,23 +21,64 @@ To use the failover feature, do the following:
 * In Anypoint MQ:
 .. Create or modify a queue and xref:mq::mq-failover.adoc#enable-failover[enable failover on the queue].
 
+
 [[enable-failover-feature]]
 == Enable Failover in Your App
 
 To enable the failover feature in your app, set the `cross.region.failover.feature.enabled` system property to `true`.
 This property is set to `false` by default.
 
+[NOTE]
+When the failover property is enabled, the connector handles processing messages for both the primary and failover queues.
+To prevent interruptions, the app can't publish or consume directly from the fallback queue.
+See <<consume-directly-fallback>> for information.
+
 To set this application property in Runtime Manager:
 
 . Sign in to Anypoint Platform and click *Runtime Manager* in the navigation menu.
 . Deploy a new app or change the settings for a deployed app.
-. On the *Properties* tab, specify values for the property:
+. On the *Properties* tab, specify the value for the property, for example:
 +
-image::mq::mq-set-properties-failover-enabled.png["Properties tab with failover properties specified"]
+image::mq-set-properties-failover-enabled.png["Properties tab with failover system property set to true"]
 
 Alternatively, set the property in *Run Configurations* in Anypoint Studio.
 For more information, see xref:mule-runtime::mule-app-properties-system.adoc[System Properties in Mule Apps].
 
+[[consume-directly-fallback]]
+== Process Messages Directly from the Fallback Queue 
+
+If failover is enabled for an app, Anypoint MQ Connector manages the fallback queue (with the `_fb` suffix) and prevents publish, consume, or subscribe operations on the fallback queue directly.
+If the app connects to the fallback queue directly, Anypoint MQ Connector returns an error:
+
+[source]
+----
+java.lang.IllegalArgumentException: Using fallback queue directly is not supported
+----
+
+If you disable failover on a queue (from the Anypoint MQ UI) and restart the application, the connector no longer publishes or consumes from the fallback queue.
+
+To process any messages that remain in the fallback queue after disabling failover:
+
+. Allow the app to consume messages directly from the fallback queue:
++
+.. Sign in to Anypoint Platform and click *Runtime Manager* in the navigation menu.
+.. Change the settings for a deployed app.
+.. On the *Properties* tab, set the `cross.region.failover.feature.enabled` system property to `false`, for example:
++
+image::mq-set-properties-failover-disabled.png["Properties tab with failover system property set to false"]
++
+Alternatively, set the property in *Run Configurations* in Anypoint Studio.
+For more information, see xref:mule-runtime::mule-app-properties-system.adoc[System Properties in Mule Apps].
+. Configure the app to process directly from the fallback queue:
```

Comment:
We should swap the order here.. user should first build an app where they specify the fallback queue name and then go to anypoint platform and set cross.region.failover.feature.enabled=false for this app.

So:
. Configure the app to process directly from the fallback queue:
. Allow the app to consume messages directly from the fallback queue: